v1/pipeline/stages/cd/inherit-service.yaml

```diff
@@ -0,0 +1,11 @@
+title: InheritService
+type: object
+required:
+  - stage
+properties:
+  stage:
+    type: string
+    pattern: ^[a-zA-Z_][0-9a-zA-Z_]{0,127}$
+  description:
+    desc: This is the description for InheritService
```

Comment:
this should be description

Reply:
it should be this

desc:
   description: